app/views/tours/_tour.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="col-md-9 order-sm-first">
+  <div class="title-search-tours">
+    <h3><%= t(".tours") %></h3>
+  </div>
+  <div>
+    <% @tours.each do |t1| %>
+      <% t1.tours.each do |t2| %>
```

Comment:
Sao code lại có `tours` rồi `tours` 1 lần nữa ri em, check lại các đối tượng nào

Reply:
Chỗ này em đổi tên lại cho dễ nhìn rồi ạ.
Vì trong 1 id_end(name_end: Hà Nội) thì có nhiều điểm đi và đến.
Mỗi điểm đi và đến ghép lại với nhau tạo thành 1 travelling.
1 travelling lại có nhiều tour: mỗi tour lại có nhiều thời gian khác nhau, nên em phải for 2 lần mới ra kết quả.
Lần trước em viết bằng Sql thuần thì không cần for 2 lần.

Reply:
Chỗ này em list 1 danh sách Điểm đến.  Em kích tùy ý sẽ cho ra 1 list tour kết quả với tên thành phố đã chọn. Lúc đầu sẽ show tất cả các tour.